.teamcity/_self/projects/WebApp.kt

```diff
@@ -579,7 +579,7 @@ object CheckCodeStyleBranch : BuildType({
 				# Avoid running more than 16 parallel eslint tasks as it could OOM
 				if [ "%run_full_eslint%" = "true" ] || [ "${'$'}TOTAL_FILES_TO_LINT" -gt 16 ] || [ "${'$'}TOTAL_FILES_TO_LINT" == "0" ]; then
 					echo "Linting all files"
-					yarn run eslint --format checkstyle --output-file "./checkstyle_results/eslint/results.xml" .
+					yarn run eslint --format checkstyle --output-file "./checkstyle_results/eslint/results.xml" ${'$'}(git diff --name-only --diff-filter=d refs/remotes/origin/trunk...HEAD | grep -E '(\.[jt]sx?|\.json|\.md)${'$'}'
```

Comment:
Seems like this will fail if zero files are found to be linted:

![Screenshot 2025-03-07 at 11 23 26](https://github.com/user-attachments/assets/03a209d8-610a-4d11-8c4a-0ca191fb1b13)

Reply:
Updated the approach to take into account other factors. See updated PR description.